index.html

```diff
@@ -94,8 +94,14 @@
 
   ### <dfn>onconnect</dfn> attribute
 
+  The {{Serial/onconnect}} attribute is an Event handler IDL attribute for the
```

Comment:
Is there a reference we can use for "event handler IDL attribute"? Why is "Event" capitalized?

Reply:
Fixed in https://github.com/reillyeon/serial/pull/5/commits/099e40309d28cd737c74f9787eef65289a5029ba